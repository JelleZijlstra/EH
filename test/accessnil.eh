#!/usr/bin/ehi
include '../lib/exception.eh'

rescue () => (foo->2 = 0)
rescue () => (echo foo->3)
rescue () => (foo->bar = 0)
rescue () => (echo foo->baz)
