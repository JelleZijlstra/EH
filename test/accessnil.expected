%%stdout%%
Unknown member foo in object of type Function: func: -> (user code)
Unknown member foo in object of type Function: func: -> (user code)
Unknown member bar in object of type Function: func: -> (user code)
Unknown member foo in object of type Function: func: -> (user code)
%%stderr%%
